pkg/platform/aws/aws.go

```diff
@@ -171,6 +177,12 @@ func createTerraformConfigFile(cfg *config, terraformRootDir string) error {
 		return errors.Wrapf(err, "failed to marshal CLC snippets")
 	}
 
+	// configure oidc flags and set it to KubeAPIServerExtraFlags.
+	if cfg.Oidc != nil {
+		oidcFlags, _ := cfg.Oidc.ToKubeAPIServerFlags(cfg.clusterDomain())
+		cfg.KubeAPIServerExtraFlags = oidcFlags
```

Comment:
Perhaps we should `append` here, in case we need to add more flags in the future?

Reply:
as of now this is not possible as Initialize is called in cli/cmd/cluster.go and again in Apply which duplicates the values. Added this comment as a TODO.